data/remnant outfits.txt

```diff
@@ -343,6 +343,23 @@ outfit "Quantum Key Stone"
 
 
 
+# Scanner:
+
+outfit "Salvage Scanner"
+	category "Systems"
+	licenses
+		Remnant
+	cost 410000
+	thumbnail "outfit/salvage scanner"
+	"mass" 7
+	"outfit space" -7
+	"outfit scan power" 13
+	"outfit scan speed" 1
+	"tactical scan power" 84
+	description "When the Remnant unraveled the secrets of the alien point defence turrets that guarded the vaults they found on Aventine, they also deciphered the mechanisms that guided the ancient weapons. With a little work they transformed those guidance systems into the fundamental tools that supports their aggressive pursuit of new technology."
```

Comment:
This entire paragraph feels a bit clunky to me. Specifically:

1. the first half sentence (everything until `Aventine`) is basically a giant string of "which"s, and there's another one right after `mechanisms`
2. `With a little work` is... i can't think of a better wording, but i get the feeling there is one. With some tweaking? research?
3. A salvage scanner is technically a tool, but more of an outfit. and it doesn't really strike me as fundamental; the role it plays for the remnant is important, but it's not a fundamental tool or piece of equipment.
4. I stumbled over `agressive pursuit`, too. They're indeed pursuing new technologies, but they're not particularly agressive about it. The only context in can think of in which the remnant actively raided other alien species is the Inhibitor description (`"We aren't going home until we steal that weapon."`). So: While they are very invested in R&D, the efforts they make as well as the reason they want new tech are primarily defensive.

I'd probably ignore one such flaw, but they appear en masse in this description.

Reply:
Thanks. I tried editing it and ended up with this:

"When the Remnant unraveled the alien point defence turrets that guarded the vaults on Aventine, they also deciphered the mechanisms that guided the ancient weapons. After much research they transformed those guidance systems into important tools for observing alien ships."

Reply:
Upon reflection, how about this:
"When the Remnant unraveled the alien point defence turrets that guarded the vaults on Aventine, they also deciphered the mechanisms that guided the ancient weapons. After much research they transformed those guidance systems into long-range observation systems."

Reply:
that sounds a lot better. i assume you will combine this with the suggestion above (https://github.com/endless-sky/endless-sky/pull/4211#discussion_r266257960)

Reply:
Modified it a bit more. And yes, the previous comment and this one are more or less rolled together at this point. How does this sound?

"When the Remnant unraveled the alien point defence turrets that guarded the vaults on Aventine, they also deciphered the mechanisms that guided the ancient weapons. After much research they transformed those guidance systems into important tools for analyzing alien ships."

Reply:
i'm still stumbling over the repeated "that" in the first sentence. the second one could just be "their guidance systems". and "into important tools" also just feels... *off* to me.

@Amazinite @Janaszar @Pointedstick y'all muricans are better at this than i am - does this bother you?

Reply:
I think I've fixed it. @MCOfficer How's this? Down to only a single "that" now.

"When the Remnant unraveled the alien point defence turrets guarding the vaults on Aventine, they also deciphered the mechanisms that guided the ancient weapons. After much research they transformed those guidance systems into important tools for analyzing alien ships."